index.html

```diff
@@ -7,10 +7,43 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Responsive Cake webpage</title>
     <!-- Add a link to your css file here -->
+    <link rel="stylesheet" href="style.css">
 </head>
 
 <body>
     <!-- Add your markup here -->
+    <div class="contener">
+        <div class="header">
+            <img src="img/icon.png" alt="icon_cake">
+            <img src="img/hum_bourger_icon.png" alt="menu">
+            <h1>The best cakes in town <br>deliverd to your door</h1>
+        </div>
+        <div class="artical">
```

Comment:
Also, the div that has a class of "article", you should use article instead of the div, because it is semantically discriptive